MH4U Dex/DamageZonesCollectionViewController.m

```diff
@@ -0,0 +1,92 @@
+//
+//  DamageZonesCollectionViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/9/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "DamageZonesCollectionViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "DamageZoneCell.h"
+#import "DamageZone.h"
+
+@interface DamageZonesCollectionViewController ()
+
+@property (weak, nonatomic) IBOutlet UICollectionViewFlowLayout *flowLayout;
+@property (nonatomic, strong) NSArray *damageZones;
```

Comment:
I always prefer having atomic/nonatomic before strong/weak, but generating IBOutlets via interface builder always sets them up the other way.  I forgot to swap this one around.